src/chat/components/camera/VideoPlayer.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useState, useRef } from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+  Image,
+  Dimensions,
+} from 'react-native';
+import Video, {
+  OnLoadData,
+  OnProgressData,
+  VideoRef,
+} from 'react-native-video';
+import { formatTime } from '../../../utilities';
+import CustomSlider from '../slider';
+
+const { height, width } = Dimensions.get('window');
+interface VideoPlayerProps {
+  videoUri: string;
+  sliderCustom?: (
```

Comment:
I think this prop should be `customSlider` or `renderCustomSlider`

Reply:
ok I updated